DayrSuryan/DSEthiop10.xml

```diff
@@ -380,26 +378,35 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                     <handNote xml:id="h1" script="Ethiopic">
                         <locus target="#1v"/>
                           <seg type="script"/>
-                          <note>The crude hand of the added beginning of the text
-                            on <ref target="#1v"/>.</note>
+                          <note>Crude later hand of the incipit of <ref target="#ms_i1"/> and
+                          <ref target="#a2"/> on <ref target="#a1"/>.</note>
                       </handNote>
 
                      <handNote xml:id="h2" script="Ethiopic">
 		                     <locus from="1v" to="5v"/>
                            <locus from="134v" to="ir"/>
                          <seg type="script"/>
+                         <note>Unnamed scribe</note>
                        </handNote>
 
 
                      <handNote xml:id="h3" script="Ethiopic">
-		                     <locus from="11r" to="109v"/>
+		                     <locus from="11r" to="53v"/>
                         <seg type="script"/>
+                        <note>Unnamed scribe</note>
                      </handNote>
 
-
                      <handNote xml:id="h4" script="Ethiopic">
-		                     <locus from="110r" to="134r"/>
+		                     <locus from="53v" to="91v"/>
+                        <seg type="script"/>
+                        <note>Hayle Maryam</note>
```

Comment:
Instead of the `<note>`, you can just give the persName with role="scribe"

Reply:
Yes, I am still doing this field.